app/frontend/src/components/views/project-board-view/ProjectBoardView.tsx

```diff
@@ -0,0 +1,242 @@
+    taskMoved({
+      ...updateParams,
+      prevStatusId,
+    });
+    // Update on server
+    changeTaskStatusFx(updateParams);
+  };
+
+  const createNewTask = (statusId: string, text: string): void => {
+    if (!currentProject) {
+      return;
+    }
+    const orderScore = tasksList.length ? (tasksList[0].orderScore + 1) : 1;
+    const taskData: CreateTaskMessagePayload = {
+      text: prepareTaskContent(text),
+      projectId: currentProject._id,
+      orderScore,
+    };
+
+    if (statusId !== UNSORTED_COLUMN_ID) {
+      taskData.statusId = statusId;
+    }
+    createTaskFx(taskData);
+  };
+
+  const handleCardClick = (task: Task): void => {
+    taskSelected(task);
+  };
+
+  return (
+    <Container>
+      <Outlet />
+      <DragDropContext onDragEnd={ onDragEnd }>
+        { getColumnsData(tasksList, currentProject?.taskStatuses).map(column => (
+          <ColumnStyled key={ column.status._id }>
+            <ColumnLabel>
+              { column.status.label }
+              <CountLabel>
+                { column.tasks.length }
+              </CountLabel>
+            </ColumnLabel>
+            <Droppable droppableId={ column.status._id } >
+              { provided => (
+                <div
+                  { ...provided.droppableProps }
+                  ref={ provided.innerRef }>
+                  { column.tasks
+                    .map((task, index) => (
+                      <Draggable
+                        draggableId={ task._id }
+                        index={ index }
+                        key={ task._id }
+                        isDragDisabled={ !currentProject }
+                      >
+                        { draggableProvided => (
+                          <StyledCardLink
+                            { ...draggableProvided.draggableProps }
+                            { ...draggableProvided.dragHandleProps }
+                            to={ task._id }
+                            key={ task._id }
+                            taskTitle={ getTaskTitle(task.text) }
+                            ref={ draggableProvided.innerRef }
+                            onClick={ () => handleCardClick(task) }
+                          />
+                        )}
+                      </Draggable>
+                    ))}
+                  { provided.placeholder }
+                  <TaskInput onChange={ (value) => createNewTask(column.status._id, value) }/>
+                </div>
+              )}
+            </Droppable>
+          </ColumnStyled>
+        ))}
+      </DragDropContext>
+    </Container>
+  );
+};
+
+/**
+ * Styled component for displaying count of tasks in column
+ */
+const CountLabel = styled.span`
+  font-size: 12px;
+  font-weight: 600;
+  color: var(--color-text-secondary);
+  margin-left: 6px;
+  vertical-align: top;
+  line-height: 120%;
+`;
+
+/**
+ * Styled component for displaying column label
+ */
+const ColumnLabel = styled.h2`
+  font-weight: 700;
```

Comment:
we already have [mixins](https://github.com/codex-team/task-manager/blob/main/app/frontend/src/styles/Mixins.ts) with typography styles. Please move all-new text styles to that way.